Module#04/model/src/main/java/com/epam/esm/entity/User.java

```diff
@@ -77,13 +90,14 @@ public class User implements Serializable {
      * The email address of the user.
      */
     @Column(name = "email", nullable = false)
-    @Pattern(regexp = "[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\\."
-            + "[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@"
-            + "(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\\.)+[a-z0-9]"
-            + "(?:[a-z0-9-]*[a-z0-9])?",
-            message = "{invalid.email}")
     private String email;
```

Comment:
@Email validation annotatoin can be used from javax validation library

Reply:
> @Email validation annotatoin can be used from javax validation library

> By adding the @Email annotation to the email field, it will enforce the validation of the email address format.